.github/workflows/dev.yml

```diff
@@ -33,10 +33,10 @@ jobs:
       BRANCH: ${{ github.ref_name }}
     steps:
       - uses: actions/checkout@v3
-      - name: install node 18
+      - name: install node 20
```

Comment:
sanity check, did we mean to update to v20 in this PR alongside the other changes?

Reply:
Apologies, I didn't mention this in the PR description. I will add it.

This cleaned up some missed references [when transitioning from Node 16 and 18 to Node 18 and 20 to reflect LTS status](https://github.com/CesiumGS/cesium/pull/11598).